code/modules/organs/external/_external.dm

```diff
@@ -549,18 +551,22 @@
 This function completely restores a damaged organ to perfect condition.
 */
 /obj/item/organ/external/rejuvenate(var/ignore_prosthetic_prefs)
-	damage_state = "00"
 
+	damage_state = "00"
 	status = 0
 	brute_dam = 0
 	burn_dam = 0
 	germ_level = 0
-	pain = 0
 	genetic_degradation = 0
+
 	for(var/datum/wound/wound in wounds)
 		qdel(wound)
 	number_wounds = 0
 
+	dislocated = initial(dislocated)
```

Comment:
@MistakeNot4892 Seems like you merged a fix to something I'm fixing before my fix was fixed O_o

Reply:
I think that came in with one of the GC fixes whoops